### PR TITLE
Mantis 18598: Add link to 'Active' tab of 'View Campaigns' page from 'Process queue' page

### PR DESCRIPTION
**Pull request by suelaP**
Add link to 'Active' tab of 'View Campaigns' page from 'Process queue' page

### CODE REVIEW

File: public_html/lists/admin/processqueue.php [hunk 1]
```diff
@@ -70,16 +70,18 @@
...
     echo '
   <div class="panel">
     <h2>' .s('Processing queued campaigns').'</h2>
     <div class="content">
       <div class="wrapper">
         <div id="spinner"></div>
         <div id="processqueuecontrols">';
-    echo '<a href="#" id="stopqueue" class="button">'.snbr('stop processing').'</a>';
-    echo '<a href="./?page=processqueue" id="resumequeue" class="button hidden">'.snbr('resume processing').'</a>';
+    echo '<a href="#" id="stopqueue" class="button">'.snbr('Stop processing').'</a>';
```
Comment (xh3n1) [inline public_html/lists/admin/processqueue.php]: @suelaP If you change this string, then a new string will be created called "Stop processing ." and as far as I know the translators will be notified that this new string must be translated, even though it's not new.

File: public_html/lists/admin/processqueue.php [hunk 2]
```diff
@@ -70,16 +70,18 @@
...
+    echo '<a href="./?page=processqueue" id="resumequeue" class="button hidden">'.snbr('Resume processing').'</a>';
```
Comment (xh3n1) [inline public_html/lists/admin/processqueue.php]: Also here :)

Comment (xh3n1) [reply public_html/lists/admin/processqueue.php]: I thought the same but apparently, we let translators the freedom to manage uppercase or lowercase.

Comment (mariez) [reply public_html/lists/admin/processqueue.php]: They are capitalized on translation. That is just a name of a string. But is ok to capitalize the new string of the new button. The problems with other languages is when you force the case using php or css.

**Comment by mariez** [comment]
screenshot from 2018-06-29 19-56-35

This is how it looks if you change this:
echo '<a href="./?page=messages&tab=active" id="sendingprogress" class="button">'.snbr('View progress').'</a>';

with this:
echo '<a href="./?page=messages&tab=active" id="sendingprogress" class="button btn-primary btn-lg pull-right">'.snbr('View progress').'</a>';

**Comment by suelaP** [comment]
I agree, thanks :)

**Comment by michield** [comment]
**Sorry, but this doesn't make sense.** 

The view progress button only makes sense when you process the queue on command line. When the queue is processed in the Browser, going to "view progress" would stop the queue from being processed.

We should not add a link to "view progress".

**Pull request closed without merging**